config/application.rb

```diff
@@ -56,5 +57,7 @@ class Application < Rails::Application
     config.autoload_paths << Rails.root.join('lib')
 
     config.active_job.queue_adapter = :sidekiq
+
+    config.middleware.insert_before 0, SameSiteSecurity::Middleware
```

Comment:
I think that could be because autoload paths (including `/lib`) are not yet loaded in application.rb.
Anyway I created a new the directory to comply with the module/class standard structure.